src/OnlineSales/Entities/BaseEntity.cs

```diff
@@ -13,15 +13,13 @@ public class BaseEntity : BaseEntityWithId
     [Required]
     public DateTime CreatedAt { get; set; }
 
-    [JsonIgnore]
     public string? CreatedByIp { get; set; } = string.Empty;
 
     [JsonIgnore]
     public string? CreatedByUserAgent { get; set; } = string.Empty;
```

Comment:
CreatedByUserAgent and UpdatedByUserAgent should also be a part of change_log.data